internal/ingress/store/fake_store.go

```diff
@@ -140,7 +140,7 @@ func NewFakeStore(
 
 			KnativeIngress: knativeIngressStore,
 		},
-		isValidIngresClass: annotations.IngressClassValidatorFuncFromObjectMeta("kong"),
+		isValidIngresClass: annotations.IngressClassValidatorFuncFromObjectMeta("kong", false),
```

Comment:
Can you make this configurable and tweak `NewFakeStore` to pass this in?

Reply:
Sure!